python/lsst/pipe/tasks/postprocess.py

```diff
@@ -607,18 +604,28 @@ class TransformObjectCatalogConfig(TransformCatalogBaseConfig):
         default="deep",
         doc="Name of coadd"
     )
+    # TODO: remove in DM-27177
     filterMap = pexConfig.DictField(
         keytype=str,
         itemtype=str,
         default={},
         doc=("Dictionary mapping full filter name to short one for column name munging."
              "These filters determine the output columns no matter what filters the "
-             "input data actually contain.")
+             "input data actually contain."),
+        deprecated=("Coadds are now identified by the band, so this transform is unused."
+                    "Will be removed after v22.")
+    )
+    outputBands = pexConfig.ListField(
+        dtype=str,
+        default=[],
+        doc=("These bands and only these bands will appear in the output,"
+             " NaN-filled if the input does not include them."
+             " If an empty list, then just send bands from the input to the output.")
```

Comment:
I don't like the special-casing of how `[]` means the _opposite_ of the general spec. Maybe use `None` as the special value instead? I think it would be less confusing.

Reply:
I also suggest rewording "then just send bands from the input to the output" to "then use all bands found in the input" (assuming that's what you meant).